pipeline/generate-preview-csharp-package.yaml

```diff
@@ -32,7 +32,16 @@ spec:
       source /workspace/source/.jx/variables.sh
       cd ./registry
       export PREVIEW_TIMESTAMP=$(date +%s)
-      /CreateCsharpPackage.sh http://$(cat ../pod_ip)/swagger.json 0.0.$PULL_NUMBER-preview-$PREVIEW_TIMESTAMP Mqube.$SwaggerServiceName.Client $REPO_OWNER $REPO_NAME
+      
+      SWAG_PATH=$(find /workspace/source -type f -name "swagger.json")
+      if test $SWAG_PATH ; then
+        echo Swagger docs found in repository
+      else
+        SWAG_PATH=http://$(cat ../pod_ip):5000/swagger/v1/swagger.json
```

Comment:
Here is swagger.json which is okay, but we need use port 80 here as well (just without :5000). Previous configuration was fine, you can check it.